src/Components/Publishing/EditorialFeature/Components/Eoy2018Artists/ArticleHeader.tsx

```diff
@@ -87,28 +103,40 @@ const GridItem = styled(Box)`
 
 const ArticleHeader = styled.div`
   height: 90vh;
+  max-height: 1000px;
   border: 3px solid ${color("purple100")};
   display: flex;
   align-items: center;
   position: relative;
+  overflow: hidden;
 `
 
 const Title = styled.h1`
+  min-height: 100%;
+  min-width: 100%;
+  padding: 20px;
   display: flex;
   flex-direction: column;
-  height: 100%;
   justify-content: space-between;
-  width: 100%;
-  padding: 20px;
   ${unica("s100")};
 `
 
-const TitleBlock = styled.span`
+// TODO: min-max font size responsive to vp
+// font-size: calc([minimum size] + ([maximum size] - [minimum size]) * ((100vw - [minimum viewport width]) / ([maximum viewport width] - [minimum viewport width])))
+const TitleBlock = styled.span<{ textAlign?: string }>`
   display: block;
   font-size: 20vh;
   line-height: initial;
 
   &:nth-child(2) {
     text-align: right;
   }
+  ${props =>
+    props.textAlign &&
+    `
+    text-align: ${props.textAlign};
+  `};
```

Comment:
If you use `styled-system` you could implement the same interface with a little less code:

```jsx
import { textAlign } from 'styled-system'

const TitleBlock = styled.span<{ textAlign?: string }>`
  ${textAlign}
`
```